lib/ty.ml

```diff
@@ -124,6 +131,12 @@ type cvtop =
   | String_from_code
   | String_to_int
   | String_from_int
+  | String_to_float
+
+type naryop =
+  | AndN
+  | OrN
```

Comment:
These can only be used with logical operators so we can probably rename them to:

```suggestion
  | Logand
  | Logor
```